hermes/internal/configuration/configuration.go

```
@@ -0,0 +1,84 @@
+/*
+ *
+ *  Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package configuration
+
+import (
+	"fmt"
+	"github.com/golang-migrate/migrate/v4"
+	postgresmigrate "github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/lib/pq"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"os"
+)
+
+var initialValues = map[string]string{
+	"DB_USER":        "charlescd_hermes",
+	"DB_PASSWORD":    "charlescd_hermes",
+	"DB_HOST":        "localhost",
+	"DB_NAME":        "charlescd_hermes",
+	"DB_SSL":         "disable",
+	"DB_PORT":        "5432",
+	"ENCRYPTION_KEY": "maycon",
```

What about use only the .env file and when the variable is not found we could throw an error, or at startup check if all variables have values

---

done